### DISCUSSION TITLE
New tasks from kanban board in obsidian

### DISCUSSION THREAD

**Discussion by Dnarts** [thesamim/TickTickSync#75] [Ideas]
Hi! Kanban obsidian://show-plugin?id=obsidian-kanban

use basically tasks in markdown. And it aaalmost work fine.

buuuut. I don’t understand why it doesn’t get sync them, even if they are already in the file. But I should change kanban view to markdown, and click to every tasks marked with #tag and press space. Then all tasks get synced. 
But isn’t is strange? I would love to see it automated. 

Ideal scenario: 
I put # to kanban task and there it is, in my ticktick.
 That would be a fairytale.

**Comment by thesamim** [comment]
@Dnarts : Sorry, just now seeing this.

TickTickSync does task adding based on keyboard input. Kanban is, essentially, taking over keyboard input.

TickTickSync also examines all the files periodically for added/modified/deleted tasks. So, those tasks will get added on Sync eventually.

Suggestion: 

1. Not recommended, but you could change your sync interval time down to less than five minutes. I don't recommend this because, depending on the number of files and tasks you have in your vault, you would put a big load on your machine.
2. Turn on debug mode for TickTickSync. This will add a refresh button to the ribbon 
![image](https://github.com/thesamim/TickTickSync/assets/5315625/6f72c756-2d31-4ff1-8a46-84f8dfd761b1)
whenever add a task, you can click that to synchronize immediately.
